src/nanotron/config/config.py

```diff
@@ -174,7 +178,7 @@ class ModelArgs:
     """Arguments related to model architecture"""
 
     model_config: NanotronConfigs
-    init_method: Union[RandomInit, ExistingCheckpointInit]
+    init_method: Union[RandomInit, MambaInit, ExistingCheckpointInit]
```

Comment:
> MambaInit

If we place Mamba in /examples, then its logic should not reside in src/nanotron